fedml_api/distributed/fedseg/MyModelTrainer.py

```diff
@@ -0,0 +1,124 @@
+import logging, time
+
+import numpy as np
+import torch
+
+try:
+    from fedml_core.trainer.model_trainer import ModelTrainer
+except ImportError:
+    from FedML.fedml_core.trainer.model_trainer import ModelTrainer
+    
+from .utils import SegmentationLosses, Evaluator, LR_Scheduler, EvaluationMetricsKeeper
+
+
+class MyModelTrainer(ModelTrainer):
```

Comment:
Why is this named "MyModelTrainer"?

Reply:
Both are same currently because we do not have any customized trainer
In future, the one in FedML will be the default trainer, and anything defined in FedCV or defined by other developers will be customized trainer. So if the customized trainer isn't specified, it will just go to default.

Reply:
I see. Could you rename this Class so that it is more intuitive?

Reply:
I thought about it as well, but later decided to stick to the same convention as we have in the FedAvg folder. What you say?

Reply:
hm, still seems a little unintuitive

Reply:
We can discuss with Chaoyang later. If it is not a convention the team is thinking to follow, we can have more intuitive name. Merging these changes for now.